<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>270</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>388</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WinCom - 查找</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>370</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>370</width>
       <height>300</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>查找</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>查找目标：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>45</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>45</height>
            </size>
           </property>
           <property name="title">
            <string>方向</string>
           </property>
           <widget class="QRadioButton" name="radioUpButton">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>22</y>
              <width>47</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>向上</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="radioDownButton">
            <property name="geometry">
             <rect>
              <x>63</x>
              <y>22</y>
              <width>47</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>向下</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber"/>
         </item>
        </layout>
       </item>
       <item row="0" column="3" rowspan="3">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="findNextBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>查找下一个</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>取消</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="2">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="allChCheckBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>81</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>全词匹配</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="matchCaseCheckBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>81</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>匹配大小写</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="loopFindCheckBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>81</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>循环查找</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QComboBox" name="comboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>22</height>
          </size>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="statusLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>tabWidget</zorder>
  <zorder>statusLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
